webapps/src/main/java/org/camunda/bpm/webapp/impl/security/filter/headersec/provider/impl/ContentSecurityPolicyProvider.java

```diff
@@ -18,15 +18,29 @@
 
 import org.camunda.bpm.webapp.impl.security.filter.headersec.provider.HeaderSecurityProvider;
 
+import javax.servlet.ServletContext;
 import java.util.Map;
+import java.util.UUID;
```

Comment:
Yeah, `SecureRandom` does seem to be slow, so let's use something else.
I like the `ThreadLocalRandom`.

The article is a bit weird though because it bashes UUID for using `SecureRandom` but at the end it suggests `SecureRandom`.

Reply:
True. The example uses `SecureRandome`. However, they also say below the example:
> I can also make the SecureRandom into a ThreadLocal if I want.